hbase-http/src/main/java/org/apache/hadoop/hbase/http/HttpServer.java

```diff
@@ -770,7 +770,7 @@ protected void addDefaultApps(ContextHandlerCollection parent, final String appD
     // set up the context for "/static/*"
     ServletContextHandler staticContext = new ServletContextHandler(parent, "/static");
     staticContext.setResourceBase(appDir + "/static");
-    staticContext.addServlet(DefaultServlet.class, "/*");
+    staticContext.addServlet(AdminAuthorizedServlet.class, "/*");
```

Comment:
What does AdminAuthorizedServlet do ?

We don't want to limit the static folder to admin users.